packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Add duration to camera animate
```

Comment:
Nit: Adds
Nit: Missing period

(See https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style)